dunai-frp-bearriver/src/FRP/BearRiver/List.hs

```diff
@@ -15,7 +15,7 @@ import           FRP.BearRiver
 newtype ListSF a b = ListSF { listSF :: SF a (b, Bool, [ListSF a b]) }
 
 dlSwitch :: [ListSF a b] -> SF a [b]
```

Comment:
How could this have compiled before without a value bound to this name?

Reply:
I don't think it is compiled, it's not mentioned in the cabal file or any imports I can find. Or maybe missing implementation is just a warning and not an error?